File: app/models/employee.rb
# == Schema Information
#
# Table name: employees
#
#  id         :integer          not null, primary key
#  last_name  :text
#  name       :text
#  position   :text
#  salary     :decimal(, )
#  created_at :datetime         not null
#  updated_at :datetime         not null
#
class Employee < ApplicationRecord
end

File: frontend/src/types/employee.ts
export interface Employee {
  id: number;
  name: string;
  salary: number;
  position: string;
}
export interface CreateEmployee {
  name: string;
  last_name: string;
  position: string;
  salary: number;
}

export interface UpdateEmployee {
  id: number;
  name: string;
  last_name: string;
  position: string;
  salary: number;
}
